### PR TITLE
Update command-reference.md

### PR DESCRIPTION
**Pull request by chenzhitong**
Fix https://github.com/neo-project/neo-express/issues/290

### CODE REVIEW

**Comment by lock9** [comment]
Hi @chenzhitong ,
I've noticed you updated the installation instructions. These will work but it's not how it should be used. Neo-express _requires no installation_. It can, and should, be shipped as a standalone application. I don't think that asking people to install the .NET SDK is reasonable. They are not going to develop anything, they don't need to install the SDK. We can embed the runtime with the application.

**Comment by chenzhitong** [comment]
> Hi @chenzhitong , I've noticed you updated the installation instructions. These will work but it's not how it should be used. Neo-express _requires no installation_. It can, and should, be shipped as a standalone application. I don't think that asking people to install the .NET SDK is reasonable. We can embed the runtime in the application.

I copied the installation guide directly from the readme.md. Isn't neo-express installed through dotnet tools? And the installation guide doesn't say anything about installing the SDK.

**Comment by lock9** [comment]
>Isn't neo-express installed through dotnet tools?
>And the installation instructions doesn't say anything about installing the SDK.

`dotnet tools` is part of the .NET SDK. We can continue to distribute it using it. However, neo-express can be compiled and shipped as a standalone app that doesn't require external tools (like dotnet tools). 
It may be fine to update the installation guide now, but be aware that we need to change the distribution method.

**Comment by chenzhitong** [comment]
I'll revert the installation instructions first and then submit it after neo-express is compiled into a standalone application

**Comment by chenzhitong** [comment]
I'll revert the installation instructions first and then submit it after neo-express is compiled into a standalone application

**Comment by chenzhitong** [comment]
I'll revert the installation instructions first and then submit it after neo-express is compiled into a standalone application @lock9

File: docs/command-reference.md [hunk 1]
```diff
@@ -714,20 +699,19 @@ local file, can be used as the input for the `policy sync` command described bel
...
-  Account                   Account to pay contract invocation GAS fee
-
-Options:
+[Options]:
   -p|--password <PASSWORD>  password to use for NEP-2/NEP-6 sender
   -i|--input <INPUT>        Path to neo-express data file
   -t|--trace                Enable contract execution tracing
   -j|--json                 Output as JSON
+<Policy>: Policy to set
+                            Allowed values are: GasPerBlock, MinimumDeploymentFee, CandidateRegistrationFee,
```
Comment (lock9) [inline docs/command-reference.md:711]: There are some differences in indentation.

File: docs/command-reference.md [hunk 2]
```diff
@@ -736,19 +720,18 @@ The `policy set` command updates the current value of the specified Neo-Express
...
-  Account                   Account to pay contract invocation GAS fee
-
-Options:
+[Options]:
   -p|--password <PASSWORD>  password to use for NEP-2/NEP-6 sender
   -i|--input <INPUT>        Path to neo-express data file
   -t|--trace                Enable contract execution tracing
   -j|--json                 Output as JSON
+<Source>: Source of policy values. Must be local policy settings JSON file or the URL of Neo JSON-RPC
+                            Node
```
Comment (lock9) [inline docs/command-reference.md:732]: There are some differences in indentation.

**Review by lock9 — COMMENTED** [review]
I saw some differences in the indentation, but everything else looks fine. By the way, the next release has an 'update' command that needs to be described here. Let's leave that for later.

**Review by lock9 — APPROVED** [review]

**Comment by lock9** [comment]
Hi @Liaojinghui, can you merge this PR? Thanks

**Review by Liaojinghui — APPROVED** [review]

**Pull request merged**